hxd/Window.hl.hx

```diff
@@ -5,6 +5,12 @@ import hxd.Key in K;
 #error "You shouldn't use both -lib hlsdl and -lib hldx"
 #end
 
+enum DisplayMode {
+	Windowed;
+	Borderless;
+	Fullscreen;
+}
+

```

Comment:
Better replace with `typedef DisplayMode = sdl/dx.Window.DisplayMode` to avoid unnecessary conversion between the two.